content/docs/00-release-notes.md

```diff
@@ -73,6 +73,11 @@ Spectro Cloud Palette 3.0.0 is released with [Native Edge](/clusters/edge/native
 * Open Policy Agent version 3.9.0
 * AWS EBS  CSI version 1.10.0
 
+**Known Issues**
+
+* While deploying multiple applications on a Sandbox cluster, the first installed application gets stuck at deployment, leads to other applications within the same Sandbox cluster getting stuck.
```

Comment:
While deploying multiple applications on a Sandbox cluster, if one of the application get stuck because of some errors it can cause other deployments in the same sandbox cluster to get stuck in provisioning state.